flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/Constants.java

```diff
@@ -30,6 +34,14 @@ public class Constants {
 
     public static final String CONFIG_FILE_LOGBACK_NAME = "logback-console.xml";
     public static final String CONFIG_FILE_LOG4J_NAME = "log4j-console.properties";
+    public static final List<String> CONFIG_FILE_NAME_LIST = Arrays.asList(
+            "logback.xml",
```

Comment:
I see [https://nightlies.apache.org/flink/flink-docs-release-2.0-preview1/docs/deployment/advanced/logging/#log4j-2-property-files](https://nightlies.apache.org/flink/flink-docs-release-2.0-preview1/docs/deployment/advanced/logging/#log4j-2-property-files) ; are we missing 
`log4j-cli.properties`